### PR TITLE
Fix 104 images location yevchuk

### PR DESCRIPTION
**Pull request by r-yevchuk**

### CODE REVIEW

**Review by Andrii35 — DISMISSED** [review]
Well done!

File: server/src/main/java/com/lunchtime/LunchTimeApplication.java [hunk 1]
```diff
@@ -1,21 +1,13 @@
...
 import org.springframework.context.annotation.Bean;
-import org.springframework.context.event.ContextRefreshedEvent;
-import org.springframework.context.event.EventListener;
 import org.springframework.web.servlet.config.annotation.CorsRegistry;
+import org.springframework.web.servlet.config.annotation.ResourceHandlerRegistry;
 import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
 
-@SpringBootApplication(
-    //TODO One time will be enough
-    exclude = {SecurityAutoConfiguration.class, SecurityAutoConfiguration.class})
```
Comment (Melnyk9506) [inline server/src/main/java/com/lunchtime/LunchTimeApplication.java]: why you delete all line of auto configuration spring security ?

**Review by Melnyk9506 — COMMENTED** [review]
pleace explain ?

File: server/src/main/java/com/lunchtime/LunchTimeApplication.java [hunk 2]
```diff
@@ -35,9 +27,12 @@ public void addCorsMappings(CorsRegistry registry) {
                     // this list without creating new PR, for example for prod env, you will need only ssh
                     .allowedOrigins("http://localhost:3000");
             }
+
+            @Override
+            public void addResourceHandlers(ResourceHandlerRegistry registry) {
+                registry.addResourceHandler("/api/**")
+                    .addResourceLocations("classpath:/static/");
+            }
```
Comment (narekacy) [inline server/src/main/java/com/lunchtime/LunchTimeApplication.java]: It is Ok, but as i know (and see on **[github](https://github.com/search?p=1&q=addResourceHandlers%28ResourceHandlerRegistry+registry%29&type=Code)**), this should be in configuration class, but not in the main launch one.

File: server/src/main/java/com/lunchtime/LunchTimeApplication.java (hunk 1 above)
Comment (r-yevchuk) [reply server/src/main/java/com/lunchtime/LunchTimeApplication.java]: Ok, i will return this code. I thought we were turning off security auto config in other place.

File: server/src/main/java/com/lunchtime/LunchTimeApplication.java (hunk 2 above)
Comment (r-yevchuk) [reply server/src/main/java/com/lunchtime/LunchTimeApplication.java]: Fixed

**Review by Melnyk9506 — COMMENTED** [review]
well i understand that you want to move web configurations to another class. and I'm wondering if you've researched whether everything will work without @SpringBootApplication(exclude = { SecurityAutoConfiguration.class }) in enother class or everithing will not work like in demo fork with login

**Review by Andrii35 — DISMISSED** [review]
It works well

**Comment by narekacy** [comment]
Server starts ok, but when I want to see all Events, i get this in console:
image

... and this in logs:
image

**Comment by r-yevchuk** [comment]
> Server starts ok, but when I want to see all Events, i get this in console:
> image
> 
> ... and this in logs:
> image

Did you clear db when start app? On my pc it works fine

**Review by narekacy — DISMISSED** [review]
Nice!

**Review by narekacy — APPROVED** [review]
Everything is in working condition

**Review by Melnyk9506 — APPROVED** [review]

**Review by Andrii35 — APPROVED** [review]

**Pull request merged**